flocker/acceptance/testtools.py

```diff
@@ -69,8 +69,10 @@ def create_application(name, image, ports=frozenset(), volume=None,
     """
     Instantiate an ``Application`` with the supplied parameters and return it.
     """
+    if ':' not in image:
```

Comment:
`DockerImage.from_string` already does this parsing so this should never have been necessary. Double check and if not create an issue for fixing that.